README.markdown

```diff
@@ -347,3 +347,15 @@ If setting the `default` alias does not establish the node version in new shells
 [3]: https://travis-ci.org/creationix/nvm
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+## Uninstall
+
+```
+rm -rf $NVM_DIR ~/.npm ~/.bower && \
```

Comment:
why would you remove `~/.npm` and/or `~/.bower`? These dirs aren't part of nvm, they're part of npm and bower, respectively.